src/main/java/com/epam/ta/reportportal/ws/model/FinishExecutionRQ.java

```diff
@@ -48,8 +48,8 @@
 	private Date endTime;
 
 	@JsonProperty(value = "status")
-	@In(allowedValues = { "passed", "failed", "stopped", "skipped", "interrupted", "cancelled" })
-	@ApiModelProperty(allowableValues = "PASSED, FAILED, STOPPED, SKIPPED, RESETED, CANCELLED")
+	@In(allowedValues = { "passed", "failed", "stopped", "skipped", "interrupted", "cancelled", "info", "warn" })
+	@ApiModelProperty(allowableValues = "PASSED, FAILED, STOPPED, SKIPPED, RESETED, CANCELLED, INFO, WARN")
```

Comment:
What was a reason of having status 'RESETED' and not having 'interrupted' here? I thought these two lines should be equal.

Reply:
fixed

Reply:
OK